src/summary/cards/summary-card-row.component.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { match, Link } from "react-router-dom";
```

Comment:
Bummer we have to thread this `match` thing through everywhere

Reply:
We probably could switch to [`useLocation`](https://reacttraining.com/react-router/web/api/Hooks/uselocation) and then it wouldn't have to be plumbed through.

However, I also just don't get very bothered by prop plumbing. I actually far prefer it over less explicit ways a lot of times. Switching to `useLocation` would be a good idea here, though.

Reply:
We probably could switch to useLocation and then it wouldn't have to be plumbed through.

However, I also just don't get very bothered by prop plumbing. I actually far prefer it over less explicit ways a lot of times. Switching to useLocation would be a good idea here, though.